docs/dev/RampUpExample.md

```diff
@@ -0,0 +1,204 @@
+# Introduction
+1. This is a [ramp up task](https://github.com/risingwavelabs/risingwave-operator/pull/350) for people who are not familiar with Kubebuilder. 
+2. During the development, these resources maybe helpful: [k8s ducoment](https://kubernetes.io/docs/home/), [kuberbuilder book](https://book.kubebuilder.io/), [our blog](https://www.risingwave-labs.com/blog/one-step-towards-cloud-the-risingwave-operator/).
+3. You will get familiar with some basic concepts in k8s 
+   1. How to create a yaml file to apply  
+   2. [CustomResourceDefinitions](https://book.kubebuilder.io/reference/generating-crd.html), kubebuilder
+4. Some people hope to build their own custom resources (CRD-Customize Resource Definition) like build-in resources, and then write a corresponding controller for the custom resources to launch their own declarative API. K8s provides a CRD extension method to meet the needs of users, and because this extension method is very flexible, a considerable enhancement has been made to CRD in the latest version 1.15. For users, implementing CRD extension mainly does two things:
+   1. Write CRD and deploy it to the K8s cluster
+      - The function of this step is to let K8s know that there is this resource and its structural attributes. 
+      - When the user submits the definition of the custom resource (usually YAML file definition), K8s can successfully verify the resource and create a corresponding Go struct for persistence , while triggering the tuning logic of the controller.
+
+   2. Write the Controller and deploy it to the K8s cluster. The role of this step is to implement the tuning logic.
+
+5. Kubebuilder is a tool to help us simplify these two things, and now we start to introduce the protagonist.
+
+
+# What is the Operator
+
+1. [CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) is costomized resource definition. Regardless of the abstraction concepts, for k8s, all it needs to manage are pods. 
```

Comment:
> all it needs to manage are pods.

This is true for our controller, but may not be true for other controllers. You could also imagine controller that reconciles CMs or services.